docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-5-0,Logstash 5.5.0>>
+
+[[logstash-5-5-0]]
+=== Logstash 5.5.0 Release Notes
+
+* Introduced a new feature called dead letter queues. When enabled, the dead letter queue (DLQ) 
+  will route events to a file based store when it can't be processed by certain plugins. Currently, 
```

Comment:
I think this description will be a little confusing to users, partly because it's grammatically ambiguous and partly because it's a little vague. How about saying something like this instead?

Introduced a new feature called dead letter queues. When enabled, the dead letter queue (DLQ)  feature will route events to a file-based store when the output plugin is unable to process the events. Currently, the Elasticsearch output is the only plugin that uses this feature to store "dead" events that result from Elasticsearch mapping errors.

[Does this mean that only mapping errors get routed to the queue? Your writeup and my edit suggests that.]